### PR TITLE
Add documentation about archiving feedstocks

### PR DESCRIPTION
**Pull request by nicoddemus**
As per https://github.com/conda-forge/pytest-pep8-feedstock/issues/8#issuecomment-551866353

cc @CJ-Wright

### CODE REVIEW

File: src/maintainer/updating_pkgs.rst [hunk 1]
```diff
@@ -191,6 +191,17 @@ Removing broken packages
 
 Sometimes mistakes happen and a broken package ends up being uploaded to the conda-forge channel. In this case, core can help you (see :ref:`fix_broken_packages`)
 
+
+Archiving feedstocks
+====================
+
+If a package is no longer maintained and/or new releases of that package are not expected in the future, ``conda-forge`` will eventually decide to *archive*
+the repository. An archived repository can no longer accept PRs and issues, which prevents people and ``regro-cf-autotick-bot`` to update the
```
Comment (CJ-Wright) [inline src/maintainer/updating_pkgs.rst]: ...from updating the...

File: src/maintainer/updating_pkgs.rst [hunk 2]
```diff
@@ -191,6 +191,17 @@ Removing broken packages
 
 Sometimes mistakes happen and a broken package ends up being uploaded to the conda-forge channel. In this case, core can help you (see :ref:`fix_broken_packages`)
 
+
+Archiving feedstocks
+====================
+
+If a package is no longer maintained and/or new releases of that package are not expected in the future, ``conda-forge`` will eventually decide to *archive*
```
Comment (CJ-Wright) [inline src/maintainer/updating_pkgs.rst]: I would remove the bit about not expected in the future.

**Comment by nicoddemus** [comment]
Done, thanks for the review!

File: src/maintainer/updating_pkgs.rst [hunk 3]
```diff
@@ -191,6 +191,17 @@ Removing broken packages
 
 Sometimes mistakes happen and a broken package ends up being uploaded to the conda-forge channel. In this case, core can help you (see :ref:`fix_broken_packages`)
 
+
+Archiving feedstocks
+====================
+
+If a package is no longer maintained ``conda-forge`` will *archive*
+the repository. An archived repository can no longer accept PRs and issues, which prevents people and ``regro-cf-autotick-bot`` from updating the
+package (an example would be to re-render the feedstock to support new Python versions).
```
Comment (CJ-Wright) [inline src/maintainer/updating_pkgs.rst]:
```suggestion
package (an example would be to re-render the feedstock to support new Python versions). Note that this **does not** remove the existing packages, those will still be available.
```

Comment (nicoddemus) [reply src/maintainer/updating_pkgs.rst]: Done, thanks

**Pull request merged**